src/test/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActorTest.scala

```diff
@@ -478,5 +479,50 @@ class TaskLauncherActorTest extends AkkaUnitTest {
         verifyClean()
       }
     }
+
+    "reschedule instance on provision timeout" in new Fixture {
+      Given("a provisioned instance")
+      val scheduledInstance = Instance.scheduled(f.app)
+      val provisionedInstance = Instance.scheduled(f.app)
```

Comment:
I hate using copy here, because then we create instances in a states that just does not reflect reality. E.g. here we have provisioned instance with no task

Reply:
No sure what you mean. This is how provisioned instances are created. We use the same fixtures in other tests.

Reply:
✅